README.md

```diff
@@ -14,44 +14,64 @@ In this project, we have created a Customer Resource along with its subordinate
 
 ## Running the service locally
 
-To run the service, please use the command `flask run`. The service is available at localhost: `http://127.0.0.1:8000`
+To run the service, please use the command `honcho start`. The service is available at localhost: `http://127.0.0.1:8080`
```

Comment:
Can you also add a section for the Kubernetes/IBM Cloud hosting of our service - Dev URL and Prod URL?
The service is already running there, so technically it's section: 'Using the service on Cloud/Kubernetes'

Reply:
Done